omod/src/main/webapp/resources/css/ugandaemr-poc-styles.css

```diff
@@ -9,11 +9,11 @@ font-weight: normal;
 }
 
 #99030{
```

Comment:
@semujju What is this class for its not clear

Reply:
This class had been removed